src/main/scala/cromwell/engine/backend/CromwellBackend.scala

```diff
@@ -0,0 +1,41 @@
+package cromwell.engine.backend
+
+import akka.actor.ActorSystem
+import org.slf4j.LoggerFactory
+import Backend.BackendyString
+
+/**
+  * Provides a global singleton access to the instantiated backend. This isn't bulletproof and can lead to
+  * exceptions being thrown but the way the system utilizes it it shouldn't be a problem (famous last words).
+  * So far the only real oddities I've seen are from the unit tests when they were trying to go straight to the
+  * interior of the system.
+  *
+  * Also we expect nearly everything to change about Backend in the next couple of months due to Intel's
```

Comment:
This whole comment was heavily influenced by me having talked at length in person w/ Miguel a bit more generally on this topic just minutes before making some edits. It should be reworded heavily. 

Anyways, the key points were: a) We expect in the coming months that cromwell will have backend*s* not backend. This means that every place where our code expects a single system-wide backend will need to change. There were only a handful of reasonable solutions to this problem in a world where we're still monolithically backended and this was the path of least resistance - however we don't know how things will be changed in order to support the multiple backends so path of least resistance seemed like a fine thing to do.

On the testing comment, I shouldn't have made that here, it was just still fresh on my mind. What I'd said was that this singleton crap was perfectly AOK in the real system but because tests latch on to arbitrary points of the system there were a lot of failures I had to deal with. Considering there are no failures now, the point is moot